### DISCUSSION TITLE
Got faild to create-react-app

### DISCUSSION THREAD

**Discussion by abenezerGithub** [termux/termux-packages#8124] [Q&A]
Am Got faild to create react app

**Comment by thunder-coding** [comment]
We can't help you with anything unless you share us your logs and command and what you are trying to achieve and how. Please provide the following:-
- Your command
- Your logs

**Reply by abenezerGithub** [reply]
What I use is 

Pkg i nodejs
npm create-react-app my-app

And it process so long and it say faild
Abort instalation b/c of npm it self.
Actually the phone unroot that's why this happening on me?
